src/consumers.ts

```diff
@@ -5,7 +5,11 @@
 'use strict';
 import { IDependency } from './collector';
 import { get_range } from './utils';
-import { Diagnostic, DiagnosticSeverity, CodeAction } from 'vscode-languageserver'
+import { Diagnostic, DiagnosticSeverity, CodeAction, CodeActionKind, DocumentUri } from 'vscode-languageserver'
+
+/* Count total # of Public and Private Vulnerability */
+let Vul_public = 0;
+let Vul_private = 0;
```

Comment:
Don't use global variables. You could get public & private count by iterating over `diagnostics` array.

Reply:
Not sure with `diagnostics` array. But i will give a try for sure.